radis/lbl/broadening.py

```diff
@@ -565,7 +565,7 @@ def voigt_lineshape(w_centered, hwhm_lorentz, hwhm_voigt, jit=True):
     wv = 2 * hwhm_voigt  # HWHM > FWHM
 
     if jit:
-        lineshape = _whiting_jit(w_centered, wl, wv)
+        lineshape = _whiting_jit(w_centered, wl.astype("float64"), wv.astype("float64"))
```

Comment:
If you're doing this then you're loosing the memory gained earlier. Is this because @jit expecteds float64? Then we'd need to update the definition in @jit.

Reply:
Yes, there was a conflict in the datatypes of `w_centered(float64)`, `wl(float32)` and `wv(float32)`. I did this because of that.